### ISSUE TITLE
How to change the font when preview?

### ISSUE THREAD

**Issue by junjiemars** [suan/instant-markdown-d#36]
There are any way to change the font when preview?

Any helps, thanks.

**Comment by JobLeonard** [comment]
It would be nice if we could just edit the CSS of the MD output somewhere in a straightforward manner, without worrying about interfering with the installation. I like my on-screen markdown with light text on a dark background, for example.

**Comment by Kevin-Mok** [comment]
Yeah, there seems to be [a dark theme](https://github.com/suan/instant-markdown-d/tree/master/css/themes/darkness), but I'm not sure how to get that working.
